test/gtest/dumpTensorTest.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
```

Comment:
Please use only lowercase letters and underscore in filenames. Please prepend test name with `debug_` as it tests debugging functionality.